Shared/Mobile-Rounds.ViewModels/Regular/Region/RegionModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Mobile_Rounds.ViewModels.Shared;
+using Mobile_Rounds.ViewModels.Shared.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mobile_Rounds.ViewModels.Regular.Region
+{
+    public class RegionModel : NotificationBase
+    {
+        public string Name { get; set; }
+
+        public AsyncCommand Navigate { get; private set; }
+
+        public RegionModel()
+        {
+            this.Navigate = new AsyncCommand((obj) =>
+            {
+                BaseViewModel.Navigator.Navigate(Shared.Navigation.NavigationType.StationSelect);
+            });
```

Comment:
on that same token, it's been bugging me that i can't figure out how to pass the name of parent screens to the breadcrumbs that are initialized in the children